### ISSUE TITLE
Add `@since` notation to block-library PHP files

### ISSUE THREAD

**Issue by tellthemachines** [WordPress/gutenberg#52720]
## What problem does this address?

The block library PHP files are automatically synced to core with the package updates, so they should be made core-ready in Gutenberg.

One thing that's missing across the several files is `@since` notation on the functions. This issue tracks adding it in those files where needed.

**Comment by costdev** [comment]
For reference: Here is [the PHP Documentation standards section](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#since-section-changelogs) relating to `@since` tags, when they should be added, and such.

**Comment by ramonjd** [comment]
I've added a few here: https://github.com/WordPress/gutenberg/pull/52820

In no way exhaustive

**Comment by ndiego** [comment]
@ramonjd @tellthemachines is there still work to do on this for 6.3?

**Comment by tellthemachines** [comment]
It's unlikely we'll be doing much more here since RC3 is in a few days. Happy for it to be moved to 6.4!

**Comment by ramonjd** [comment]
> is there still work to do on this for 6.3?

Thanks for the question. Yeah, it requires a bit of time to pick through. We might not have caught them all, but the major stuff has been done I think.

**Comment by gziolo** [comment]
Should we enforce usage of `@since` through the linter in these files? @anton-vlasenko is working on similar enhancements that will make these files compatible with WP Core.

**Comment by tellthemachines** [comment]
Yeah that would be great! We still have to retroactively add it to all the functions though 😅

**Comment by youknowriad** [comment]
Every core-synced block PHP file now carries @since; the only files without it are experimental blocks that aren't shipped to core. Closing as done.
